Assets/Scripts/Game/WeaponManager.cs

```diff
@@ -646,6 +646,16 @@ private void WeaponDamage(FPSWeapon weapon, out bool hitEnemy)
                         hitEnemy = true;
                     }
                 }
+
+                // Check if hit a mobile NPC
+                MobilePersonNPC mobileNpc = hit.transform.GetComponent<MobilePersonNPC>();
+                if (mobileNpc)
+                {
+                    // TODO: Create blood splash.
+                    weapon.PlayHitSound();
+                    mobileNpc.Motor.gameObject.SetActive(false);
```

Comment:
Will probably need to handle this through population manager so it can recycle the NPC. I'm happy to work on this if you like, and add the blood splash. I've added this to my queue.